cal.go

```diff
@@ -255,6 +258,25 @@ func (c *Calendar) WorkdaysFrom(start time.Time, offset int) time.Time {
 	return date
 }
 
+// WorkHoursNrInRange returns the number of working days in a range starting from the set start date to the end date,
```

Comment:
This comment doesn't seem to match the method. What is the purpose of this method? Looks like you're trying to see how many holiday hours are in the next x hours?

Reply:
Comment updated. Basically is returning the number of no working hours in a range. We have a case that is needed and it can be helpful for others too I guess. Thanks for the comments